thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/api/AnomalyFeedback.java

```diff
@@ -0,0 +1,109 @@
+package com.linkedin.thirdeye.detector.api;
+
+import com.linkedin.thirdeye.constant.AnomalyFeedbackType;
+import com.linkedin.thirdeye.constant.FeedbackStatus;
+import java.io.Serializable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "anomaly_feedback")
+public class AnomalyFeedback implements Serializable{
+  private static final long serialVersionUID = 1L;
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+
+  @Column(name = "feedback", nullable = false)
+  @Enumerated(EnumType.STRING)
+  private AnomalyFeedbackType feedback;
+
+  @Column(name = "status", nullable = false)
+  @Enumerated(EnumType.STRING)
+  private FeedbackStatus status;
+
+  @Column(name = "comment", length = 500)
+  private String comment;
+
+  public long getId() {
+    return id;
+  }
+
+  public void setId(long id) {
+    this.id = id;
+  }
+
+  public AnomalyFeedbackType getFeedback() {
+    return feedback;
+  }
+
+  public void setFeedback(AnomalyFeedbackType feedback) {
+    this.feedback = feedback;
+  }
+
+  public FeedbackStatus getStatus() {
+    return status;
+  }
+
+  public void setStatus(FeedbackStatus status) {
+    this.status = status;
+  }
+
+  public String getComment() {
+    return comment;
+  }
+
+  public void setComment(String comment) {
+    this.comment = comment;
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+
+    AnomalyFeedback that = (AnomalyFeedback) o;
+
+    if (id != that.id) {
```

Comment:
Chain Objects.equals calls instead (see AnomalyFunctionSpec.java for an example).